src/lib/cms.ts

```diff
@@ -67,7 +67,7 @@ function proxiedRivetClient(productSlug: ProductSlug) {
 }
 
 export default instance
-export { client, proxiedRivetClient }
+export { productConfig, client, proxiedRivetClient }
```

Comment:
Exports `productConfig`, which is a dictionary of `ProductData`, as it's needed in `pages/_error.tsx`'s `getServerSideProps` to render the product-specific top navigation (where possible).